apps/astarte_appengine_api/lib/astarte_appengine_api/device/aliases.ex

```diff
@@ -124,7 +124,7 @@ defmodule Astarte.AppEngine.API.Device.Aliases do
   defp apply_delete(%Changeset{valid?: false} = changeset, _delete_aliases),
     do: changeset
 
-  defp apply_delete(changeset, delete_aliases) when length(delete_aliases) == 0,
+  defp apply_delete(changeset, delete_aliases) when delete_aliases == [],
```

Comment:
(this is related to all similar funcitons)

in general, guards add quite a bit of verbosity. sometimes it makes things clearer, but in this instance I think the shorter version is just as readable


```suggestion
  defp apply_delete(changeset, [] = _delete_aliases),
```

feel free to _not_ add the alias, I think it helps clear what everything is in more complex scenarios but it's not something we always (or even often) do